temp/legacy-code/src/main/java/com/ivy/legacy/legacy/ui/theme/modal/DeleteModal.kt

```diff
@@ -134,14 +138,14 @@ fun BoxWithConstraintsScope.DeleteConfirmationModal(
         Spacer(Modifier.height(12.dp))
 
         val view = LocalView.current
-        val focusRequester = remember { FocusRequester() }
+        val focusRequester = FocusRequester()
```

Comment:
I think we should `remember` the focus requester to make it work. Why do we make this change?